File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/PatchFeatureFlagExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class PatchFeatureFlagExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        var patch1 = new PatchOperation(
            op: "replace",
            path: "/description"
        );

        var patch = new List<PatchOperation>
        {
            patch1,
        };

        var patchWithComment = new PatchWithComment(
            patch: patch
        );

        try
        {
            var response = new FeatureFlagsApi(config).PatchFeatureFlag(
                projectKey: "projectKey_string",
                featureFlagKey: "featureFlagKey_string",
                patchWithComment: patchWithComment
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling FeatureFlagsApi#PatchFeatureFlag: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/PatchFeatureFlagExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

const patch1: api.PatchOperation = {
  op: "replace",
  path: "/description",
};

const patch = [
  patch1,
];

const patchWithComment: api.PatchWithComment = {
  patch: patch,
};

new api.FeatureFlagsApi(configuration).patchFeatureFlag(
  "projectKey_string", // projectKey
  "featureFlagKey_string", // featureFlagKey
  patchWithComment,
  undefined, // ignoreConflicts
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling FeatureFlagsApi#patchFeatureFlag:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
